src/main/scala/mesosphere/marathon/state/GroupManager.scala

```diff
@@ -157,7 +162,10 @@ class GroupManager @Singleton @Inject() (
       from <- rootGroup()
       (toUnversioned, resolve) <- resolveStoreUrls(assignDynamicServicePorts(from, change(from)))
       to = GroupVersioningUtil.updateVersionInfoForChangedApps(version, from, toUnversioned)
-      _ = BeanValidation.requireValid(ModelValidation.checkGroup(to, "", PathId.empty, config.maxApps.get))
+      _ = {
+        // TODO AW: test
+        validateOrThrow(V2Group(to))(validator = V2Group.v2GroupWithConfigValidator(config))
```

Comment:
@kolloch Will do. I assigned #2426 to myself for now. @aquamatthias do you mind me trying to get rid of V2?